README.md

```diff
@@ -55,6 +59,11 @@ The script `tagandbuild.bash` tags and runs crosscompile.bash.
 
 Note - ./crosscompile.bash omits debug symbols to keep the build smaller.
 
+### Linux
+
+crosscompile currently can't target Linux, so you will need to manually build on
```

Comment:
@oxtoacart  If this is a known bug or limitation, could you please add a reference URL, an error message or a very short explanation on the reason?